web_src/less/_base.less

```
@@ -123,6 +123,7 @@
   --color-text-light-3: #a0a0a0;
   --color-box-header: #f7f7f7;
   --color-box-body: #ffffff;
+  --color-box-body-highlight: #f6f6f6;
```

```suggestion
  --color-box-body-highlight: #f8f8f8;
```

I made a edit to the patch to make it very slightly brighter, maybe you missed it.

---

I missed it, didn't refresh the page prior to my adjustments